### ISSUE TITLE
iBooks canvas doesn’t support drawImage()

### ISSUE THREAD

**Issue by dvschultz** [dvschultz/99problems#2]
iBooks supports canvas, but not `drawImage`. Example to follow.

**Comment by dvschultz** [comment]
another case of it: http://stackoverflow.com/questions/14063864/ipad-ibooks-javascript-canvas-drawimage-does-not-work

**Comment by teytag** [comment]
Yes... This case is not working iBooks on iPad and iPhone. But it works iBooks on Mac. And others...

![screen shot 2014-07-13 at 12 54 57 pm](https://cloud.githubusercontent.com/assets/6927098/3564213/88ca7d82-0a76-11e4-806d-ad743b4c88f5.png)
![screen shot 2014-07-13 at 12 56 57 pm](https://cloud.githubusercontent.com/assets/6927098/3564214/90233876-0a76-11e4-8e69-2cec4dd6206d.png)
![screen shot 2014-07-13 at 12 57 45 pm](https://cloud.githubusercontent.com/assets/6927098/3564215/9a7f83f6-0a76-11e4-83cf-5843c67a4312.png)
![screen shot 2014-07-13 at 12 58 10 pm](https://cloud.githubusercontent.com/assets/6927098/3564216/9acbc57c-0a76-11e4-837a-908d7b1be347.png)
![screen shot 2014-07-13 at 12 59 16 pm](https://cloud.githubusercontent.com/assets/6927098/3564220/9af91234-0a76-11e4-96fc-46dbc354213a.png)
![screen shot 2014-07-13 at 1 00 27 pm](https://cloud.githubusercontent.com/assets/6927098/3564217/9ad625f8-0a76-11e4-9cc0-e61b37878c6d.png)
![iphone canvas](https://cloud.githubusercontent.com/assets/6927098/3564218/9ae17aa2-0a76-11e4-8cd9-5293c2f48809.PNG)
![ipad canvas](https://cloud.githubusercontent.com/assets/6927098/3564219/9aed345a-0a76-11e4-97cb-e585d56059c3.PNG)

**Comment by shornikov** [comment]
hi. use fixed layout mode in ipad-iphone. work fine

**Comment by teytag** [comment]
Thanks for info. I will try it within FXL
